middleware/logging.go

```diff
@@ -94,6 +96,11 @@ func (l Log) Wrap(next http.Handler) http.Handler {
 
 			return
 		}
+
+		if statusCode >= 200 && statusCode < 300 && l.DisableRequestSuccessLog {
```

Comment:
Would you consider phrasing the three possibilities as `switch` cases?

Reply:
sure thing.

Reply:
Addressed by https://github.com/weaveworks/common/pull/284/commits/565676c80fb17b86d151a6863c6f18517ea0c809. WDYT?